runtime/lua/vim/_editor.lua

```diff
@@ -297,7 +300,16 @@ end
 ---       autocmd FileType c setlocal cindent
 ---     augroup END
 ---   ]])
----   vim.cmd({ cmd = 'echo', args = { '"foo"' } })
+---
+---   vim.cmd { cmd = 'echo', args = { '"foo"' } }
+---   vim.cmd.echo({ args = { '"foo"' } })
+---   vim.cmd.echo('"foo"')
+---
+---   vim.cmd { cmd = 'write', args = { "myfile.txt" }, force = true }
+---   vim.cmd.write { args = { "myfile.txt" }, force = true }
+---   vim.cmd.write {"myfile.txt", force = true })
```

Comment:
force is spelled "bang" in nvim_cmd and nvim_parse_cmd, afaict.

```suggestion
---   vim.cmd { cmd = 'write', args = { "myfile.txt" }, bang = true }
---   vim.cmd.write { args = { "myfile.txt" }, bang = true }
---   vim.cmd.write {"myfile.txt", bang = true })
```